src/types.jl

```diff
@@ -134,7 +135,7 @@ end
 # A `DateTime` that includes `TimeZone` information.
 # """
 
-struct ZonedDateTime <: TimeType
+struct ZonedDateTime <: Compat.AbstractDateTime
```

Comment:
I was originally worried about this as I thought `AbstractDateTime` was a `Union` type on 0.6 (it isn't).

Reply:
A quick style question/comment. I usually would just reference this without `Compat.` so that in the future when I drop that Compat requirement I don't have to change two separate lines. Do you have reason for explicitly using `Compat.` here?

Reply:
Only because it isn't exported from Compat, but I could do an explicit `using Compat: AbstractDateTime` if you'd prefer.

Reply:
Ah ok. I would prefer `using Compat: AbstractDateTime` if it isn't too much trouble.

Reply:
Not at all!